uPortal-soffit-connector/src/main/java/org/apereo/portal/soffit/connector/SoffitConnectorController.java

```diff
@@ -57,6 +57,10 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.portlet.bind.annotation.RenderMapping;
 
+/**
+ * @since 5.0
+ * @author drewwills
```

Comment:
That makes sense to me.  Some IDEs add them, and we have tons of them in existing .java files.